io/disk/tiobench.py

```diff
@@ -41,6 +44,12 @@ class Tiobench(Test):
     Avocado test for tiobench.
     """
 
+    def clear_dmesg(self):
+        """
+        clears the dmesg logs.
+        """
+        process.run("dmesg -C ", sudo=True)
```

Comment:
please use avocado-based `dmesg` utility 

https://github.com/avocado-framework/avocado/blob/master/avocado/utils/dmesg.py

Reply:
> please use avocado-based `dmesg` utility
> 
> https://github.com/avocado-framework/avocado/blob/master/avocado/utils/dmesg.py

Done. Thanks for pointing this.